src/osd/osd_types_fmt.h

```diff
@@ -56,9 +59,12 @@ struct fmt::formatter<chunk_info_t> {
   template <typename FormatContext>
   auto format(const chunk_info_t& ci, FormatContext& ctx)
   {
-    return fmt::format_to(ctx.out(), "(len: {} oid: {} offset: {} flags: {})",
-			  ci.length, ci.oid, ci.offset,
-			  ci.get_flag_string(ci.flags));
+    return fmt::format_to(ctx.out(),
+                          "(len: {} oid: {} offset: {} flags: {})",
+                          ci.length,
+                          ci.oid,
+                          ci.offset,
+                          ci.get_flag_string(ci.flags));
```

Comment:
Unnecessary?

Reply:
There seem to be a bunch of these mixed into behavior related changes.  Either remove them or separate them into their own commit.

Reply:
moved to a separate PR